application/src/main/java/org/togetherjava/tjbot/commands/moderation/ModerationUtils.java

```diff
@@ -376,33 +377,37 @@ record TemporaryData(Instant expiresAt, String duration) {
     /**
      * Gives out advice depending on the {@link ModerationAction} and the parameters passed into it.
      *
-     * @param action gives out a {@link ModerationAction} to be able to respond and build a message
-     *        appropriately such as {@link ModerationAction#MUTE} etc.
-     * @param message is the message that is being used to build responses, such as
-     *        {@link Guild#getName()}, and reasoning for an action.
+     * @param action the action that is being performed, such as banning a user.
+     * @param temporaryData is the message that is being used to display action duration.
+     * @param guild to retrieve the guild.
+     * @param reason for the action.
      * @return the appropriate advice.
      */
-    public static String getDmAdvice(ModerationAction action, String... message) {
-        if (message.length == 3) {
+    public static String getDmAdvice(ModerationAction action,
+            @Nullable ModerationUtils.TemporaryData temporaryData, Guild guild, String reason) {
+        if (action == ModerationAction.MUTE) {
+            String durationMessage =
+                    temporaryData == null ? "permanently" : "for " + temporaryData.duration();
             return """
-                    Hey there, sorry to tell you but unfortunately you have been %s %s from the server %s.
+                    Hey there, sorry to tell you but unfortunately you have been %s %s in the server %s.
                     To get in touch with a moderator, you can simply use the **/modmail** command here in this chat. Your message will then be forwarded and a moderator will get back to you soon :thumbsup:
```

Comment:
ur still using `/modmail` as text here (see my rework suggestion, which gets rid of that issue)